doc/ctp_install_guide.md

```diff
@@ -57,8 +57,45 @@
               ctp.sh medium medium    #execute medium twice
               ctp.sh webconsole start #start web console to view sql test results
 
-
-## 2. Install CTP as Regression Test in Linux platform
+## 2. Install CTP in Windows platform
+* ### Install JDK  

```

Comment:
Remove "Install JDK".
Since it is included in linux steps. You descibe to follow it.